docs/toc.json

```diff
@@ -0,0 +1,346 @@
+    "type": "compute",
+    "nav": [{
+      "title": "Address",
+      "type": "compute/address"
+    }, {
+      "title": "Disk",
+      "type": "compute/disk"
+    }, {
+      "title": "Firewall",
+      "type": "compute/firewall"
+    }, {
+      "title": "Network",
+      "type": "compute/network"
+    }, {
+      "title": "Operation",
+      "type": "compute/operation"
+    }, {
+      "title": "Region",
+      "type": "compute/region"
+    }, {
+      "title": "Snapshot",
+      "type": "compute/snapshot"
+    }, {
+      "title": "VM",
+      "type": "compute/vm"
+    }, {
+      "title": "Zone",
+      "type": "compute/zone"
+    }]
+  }, {
+    "title": "DNS",
+    "type": "dns",
+    "nav": [{
+      "title": "Zone",
+      "type": "dns/zone"
+    }, {
+      "title": "Record",
+      "type": "dns/record"
+    }, {
+      "title": "Change",
+      "type": "dns/change"
+    }]
+  }, {
+    "title": "Datastore",
+    "type": "datastore",
+    "nav": [{
+      "title": "Dataset",
+      "type": "datastore/dataset"
+    }, {
+      "title": "Query",
+      "type": "datastore/query"
+    }, {
+      "title": "Transaction",
+      "type": "datastore/transaction"
+    }]
+  }, {
+    "title": "Logging",
+    "type": "logging",
+    "nav": [{
+      "title": "Sink",
+      "type": "logging/sink"
+    }, {
+      "title": "Log",
+      "type": "logging/log"
+    }, {
+      "title": "Entry",
+      "type": "logging/entry"
+    }]
+  }, {
+    "title": "Prediction",
+    "type": "prediction",
+    "nav": [{
+      "title": "Model",
+      "type": "prediction/model"
+    }]
+  }, {
+    "title": "PubSub",
+    "type": "pubsub",
+    "nav": [{
+      "title": "Topic",
+      "type": "pubsub/topic"
+    }, {
+      "title": "Subscription",
+      "type": "pubsub/subscription"
+    }]
+  }, {
+    "title": "Resource",
+    "type": "resource",
+    "nav": [{
+      "title": "Project",
+      "type": "resource/project"
+    }]
+  }, {
+    "title": "Search",
+    "type": "search",
+    "nav": [{
+      "title": "Index",
+      "type": "search/index"
+    }, {
+      "title": "Document",
+      "type": "search/document"
+    }, {
+      "title": "Field",
+      "type": "search/field"
+    }]
+  }, {
+    "title": "Storage",
+    "type": "storage",
+    "nav": [{
+      "title": "Bucket",
+      "type": "storage/bucket"
+    }, {
+      "title": "File",
+      "type": "storage/file"
+    }, {
+      "title": "Channel",
+      "type": "storage/channel"
+    }]
+  }, {
+    "title": "Translate",
+    "type": "translate"
+  }],
+  "types": [{
+    "id": "gcloud",
+    "title": "Node.js",
+    "contents": "index.json"
+  }, {
+    "id": "bigquery",
+    "title": "BigQuery",
+    "contents": "bigquery/index.json"
+  }, {
+    "id": "bigquery/dataset",
+    "title": "BigQuery » Dataset",
+    "contents": "bigquery/dataset.json"
+  }, {
+    "id": "bigquery/table",
+    "title": "BigQuery » Table",
+    "contents": "bigquery/table.json"
```

Comment:
I apologize for not calling this out in the PR description, but I left them out intentionally. I'm not opposed to adding methods at all, but I am a little lost on how the deeplinking will work and felt that it might be better placed in a separate PR that came after this (Perhaps @quartzmo could help out there). I think that this PR is a necessary first step that will (hopefully) make it easier to implement.